mrp_multi_level/wizards/mrp_multi_level.py

```diff
@@ -537,6 +537,29 @@ def _mrp_initialisation(self, mrp_areas):
                 self._init_mrp_move(product_mrp_area)
         logger.info("End MRP initialisation")
 
+    def _get_qty_to_order(self, product_mrp_area, date, move_qty, onhand):
+        """compute the qty to order at a given date, for a product MRP area, given an
+        mrp.move quantity and an onhand quantity.
+
+        The method takes into account the safety stock priorization and safety
+        stock target date.
+        """
+        if (
+            product_mrp_area.mrp_area_id.priorize_safety_stock
+            and date < product_mrp_area.mrp_area_id.safety_stock_target_date
```

Comment:
```suggestion
            product_mrp_area.mrp_area_id.priorize_safety_stock
            and date
            and product_mrp_area.mrp_area_id.safety_stock_target_date
            and date < product_mrp_area.mrp_area_id.safety_stock_target_date
```

Will fix: https://github.com/OCA/manufacture/actions/runs/7100988379/job/19328179577?pr=1123#step:8:587